baseplate/core.py

```diff
@@ -317,19 +333,44 @@ def __exit__(self, exc_type, value, traceback):
         else:
             self.finish()
 
+    def make_local(self, component_name, op_name):
```

Comment:
I think there are enough differences between the two types that its own method is warranted. Child spans and local spans are more sibling types than subtypes, and I think using the boolean flag implies local spans are a specific type of child span.